protocol/messenger.go

```diff
@@ -1444,13 +1442,7 @@ func (m *Messenger) handleENSVerified(records []*ens.VerificationRecord) {
 		}
 	}
 
-	m.logger.Info("calling on contacts")
-	if m.config.onContactENSVerified != nil {
-		m.logger.Info("called on contacts")
-		response := &MessengerResponse{Contacts: contacts}
-		m.config.onContactENSVerified(response)
-	}
-
```

Comment:
I see in the PR description that this was was removed to increase readability. But is the call to `m.config.oNContactENSVerified()` not needed anymore?

Reply:
`onContactENSVerified ` was previously set to `publishMessengerResponse` in `WithENSVerificationConfig`.
Now when `PublishMessengerResponse` is accessibly from here, we just call it directly.